### PR TITLE
appsettings key SDK integrationin web.config

### PR DESCRIPTION
**Pull request by amshekar**
SDK Integration in web.config.
Added sample example for appsettings integration with SDK .
This is common use case most of the app integration will use. I did lot of analysis due to less documentation , Please merge this so which will help to other engineers those who have this scenario.
@bharaba
@afalahi @tonygambone @ivanpfeff @JeremyThycotic @CordThycotic @noahhai

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+https://medium.com/@amshekar/thycotic-pam-sdk-integration-in-appsettings-section-of-web-config-9f1e8ab88413
+https://medium.com/@amshekar/thycotic-pam-sdk-integration-with-c-rest-54bed114d7a4
```
Comment (wsmelton) [inline README.md:2]: These should be removed...

File: README.md [hunk 2]
```diff
@@ -324,6 +327,30 @@ New connection string with Secret Server SDK:
     />
   </connectionStrings>
 
+```
+Example old appsetting section Keys in web.config:
+```xml
```
Comment (wsmelton) [inline README.md:332]: A new line should be added between the ending code block and a line of text, as well as the next start of a code block for proper formatting in markdown.

Comment (wsmelton) [reply README.md:332]: I would also recommend adding a description text to point out that this is a second example of the same use case. Help make it clear to the users reading that it is just a another form of the connection details in the app settings.

File: README.md (hunk 1 above)
Comment (amshekar) [reply README.md]: Removed the medium blog references

**Comment by amshekar** [comment]
@wsmelton  did you get a time to merge the pull request

**Comment by wsmelton** [comment]
I have not had time to verify the content, but at this time this repository is no longer being maintained and current content is being migrated to the new documentation system.

**Pull request closed without merging**